x-pack/plugins/alerting/server/lib/rule_execution_status.ts

```diff
@@ -137,9 +139,18 @@ export function ruleExecutionStatusFromRaw(
   return executionStatus;
 }
 
-export const getRuleExecutionStatusPending = (lastExecutionDate: string) => ({
-  status: 'pending' as RuleExecutionStatuses,
+export const getRuleExecutionStatusPendingAttributes = (
+  lastExecutionDate: string
+): RawRule['executionStatus'] => ({
+  status: 'pending',
   lastExecutionDate,
   error: null,
   warning: null,
 });
+
+export const getRuleExecutionStatusPending = (
```

Comment:
I think you can delete this one!!!

Reply:
Oh so this is being used in the new `create_rule`